cmd/docker-app/status.go

```diff
@@ -35,6 +37,14 @@ func runStatus(dockerCli command.Cli, claimName string, opts credentialOptions)
 	if err != nil {
 		return err
 	}
+
+	if c.Bundle.Actions == nil {
+		return errors.New(`specified bundle has no "status" action`)
+	}
```

Comment:
A dereference of a nil map returns `"", false` so the checks for `Actions["status"]` is sufficient, so this can be dropped. 

(If it *was* necessary to do this it should be checked somewhere further up the call chain, since it would be common to all actions)